docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -21,15 +21,12 @@ must already be imported into the control plane MariaDB;
  ** the xref:adopting-the-image-service_{context}[Adopting the Image service] needs to be imported;
  ** the xref:migrating-ovn-data_migrating-databases[Migrating OVN data] need to be imported;
  ** the xref:adopting-the-networking-service_{context}[Adopting the Networking service] needs to be imported;
- ** the xref:adopting-the-bare-metal-provisioning-service_{context}[Adopting the Openstack Baremetal service] needs to be imported;
+ ** the xref:adopting-the-bare-metal-provisioning-service_{context}[Adopting the Openstack Baremetal service] needs to be imported; 
+//kgilliga:Need to revist this xref. Might rewrite this section anyway.
  ** Required services specific topology
-xref:pulling-the-openstack-configuration_{context}[Pulling the OpenStack configuration].
-//kgilliga: this xref should specifically point to the Get services topology specific configuration module when it's ready.
+xref:proc_retrieving-services-topology-specific-configuration_{context}[Retrieving services from a topology specific-configuration].
  ** {rhos_prev_long} services have been stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
-
-.Variables
-
-Define the shell variables and aliases used in the steps below. The values are just illustrative, use values that are correct for your environment:
+* Define the shell variables and aliases used in the steps below. The values are just illustrative, use values that are correct for your environment:
```

Comment:
Per the ISG:
Do not use [below] to refer to an earlier software version or fix level. Use "earlier". Do not use to indicate a relative location in a document, as in "the information below". Use "the information later in this section". Alternatively, you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."